.github/workflows/atfe_nightly_build_and_test.yml

```diff
@@ -65,22 +98,21 @@ jobs:
             arm-software/embedded/patches/llvm-project-perf
 
       - name: Build ${{ matrix.build_script }}
-        run: bash ./arm-software/embedded/scripts/${{ matrix.build_script }}
+        run: ./arm-software/embedded/scripts/${{ matrix.build_script }}
 
       - name: Test ${{ matrix.test_script }}
-        if: matrix.test_script != ''
```

Comment:
I believe you need to keep this line, as the overlay packages don't have a `test_script` entry.